### DISCUSSION TITLE
乌冬博客

### DISCUSSION THREAD

**Discussion by Yahoo94** [MaLuns/hexo-theme-async#74] [Show 分享展示]
{
    "name": "乌冬",
    "url": "https://hexo.yahoo94.love",
    "avatar": "https://yahoo94.love/upload/2022/11/11.png",
    "desc": "再过十年，我们还是过着今天。"
}

**Comment by MaLuns** [comment]
已添加😏

**Reply by Yahoo94** [reply]
好耶，目前还有部分东西没配置完，等最近忙完了再搞一下
